west.yml

```diff
@@ -88,7 +88,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 708c95825b0d5279620935a1356299fff5dfbc6e
+      revision: f3632589e8f8d15886df1d1070cba783413e4ca3
```

Comment:
Can you move `west.yml` update into a separate commit, or add info to this commit message making it clear what the update here is pulling in?